cockroachdb/manifests-cert-manager/scripts.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
 data:
   user-schema-bootstrap.sh: |
     #!/bin/bash
-    set -e
```

Comment:
do we really want to remove this? It could mean the script exiting with 0 even if e.g. an SQL command failed

Reply:
[unremoved](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/51/files#diff-c01384ba7fdb6490332c23ca6adcea90f6709d866b02bd1b3fe48372a837750eR23)